src/plugins/console/server/lib/spec_definitions/json/overrides/cluster.post_voting_config_exclusions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "cluster.post_voting_config_exclusions": {
+    "url_params": {
+      "wait_for_removal": {
+        "__one_of": [
```

Comment:
This doesn't seem to trigger autocomplete for me and I couldn't find an example of `__one_of` implemented for `url_params` in other endpoints. Have you seen this used before? Am I doing it wrong?

![image](https://user-images.githubusercontent.com/1238659/102675260-8f748f00-414d-11eb-9585-0ecc833ea92e.png)